lib/config/configuration.js

```diff
@@ -453,7 +453,7 @@ Configuration.prototype._processConfig = function(config) {
     }
 
     if (options.hasOwnProperty('maxErrors')) {
-        this._loadMaxError(options.maxErrors);
+        this._loadMaxError(options.maxErrors === 'null' ? null : options.maxErrors);
```

Comment:
Usually I've seen -1 used to disable this kind of things. But if we replace null with -1 it will not be backwards compatible. 

If we keep both null and also add -1 I don't think it really makes sense to have these much options and it might be confusing for the users.

How do you think is best to proceed with this?